src/index.html

```diff
@@ -16,17 +16,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="search-container search-container_big"
+      data-qa="big"
+    >
+      <label class="search__label search__label_big">
+        <input
+          class="search-bar search-bar_big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
```

Comment:
There should be an empty line between sibling form elements to add some 'air' and simplify reading. Please add an empty line between the closing `</form>` tag on line 32 and the opening `<form>` tag on line 33.